datasets/README.md

```diff
@@ -0,0 +1,9 @@
+# Datasets
+
+## Description
+
+This is the folder where datasets should be downloaded to.
+
+## Installation
+
+After downloading the .csv files from the [challenge](https://challengedata.ens.fr/challenges/28), install by running `make_npz.py`
```

Comment:
As discussed, all conversion are to be made in the benchmark repo. I will keep the dataset file in the .gitignore for ease of use, but there is no point in adding a readme.

Reply:
Alright